contrib/babelfishpg_common/src/collation.c

```diff
@@ -601,6 +634,18 @@ translate_collation(const char *collname, bool check_for_server_collation_name_g
 	return idx;
 }
 
+/* Translate BBF collation to it's closest TSQL Collation. */
+const char *
+translate_bbf_collation_to_tsql_collation(const char *collname)
+{
+	int last = TOTAL_REVERSE_COLL_TRANSLATION_COUNT - 1;
```

Comment:
aren't we missing the last collation due to count - 1?

Reply:
Oh yes! Fixed it.